lib/rules/web/ocm_console/creation_cards_page.xyaml

```diff
@@ -87,3 +93,41 @@ check_creation_cards_page_title:
     scripts:
     - command: return document.title=='Create an OpenShift cluster | Red Hat OpenShift Cluster Manager'
       expect_result: true
+check_creation_cards_page_browser_title:
```

Comment:
As I said on [PR!8653](https://github.com/openshift/cucushift/pull/8653#discussion_r671969468), please move the rule to install_page.xyaml. 

Besides, please reuse the rules if they are fit for your checkpoints.